cmd/admin-info.go

```diff
@@ -187,31 +187,35 @@ func mainAdminInfo(ctx *cli.Context) error {
 	// If the error is not nil and not unrecognizable, just print it and exit
 	fatalIf(probe.NewError(e), "Cannot get service status.")
 
-	// Construct the backend status
-	storageInfo := backendStatus{
-		Total: serverInfo.StorageInfo.Total,
-		Free:  serverInfo.StorageInfo.Free,
-	}
-	if serverInfo.StorageInfo.Backend.Type == madmin.Erasure {
-		storageInfo.Backend = xlBackend{
-			Type:         erasureType,
-			OnlineDisks:  serverInfo.StorageInfo.Backend.OnlineDisks,
-			OfflineDisks: serverInfo.StorageInfo.Backend.OfflineDisks,
+	for _, serverInfo := range serversInfo {
```

Comment:
Using a arrow signifies more of a hierarchy but here we need to display something closer to `systemctl` output IMO.

For example. 
```
● ssh.service - OpenBSD Secure Shell server
   Loaded: loaded (/lib/systemd/system/ssh.service; enabled; vendor preset: enabled)
   Active: active (running) since Fri 2017-03-24 14:00:24 PDT; 1 months 19 days ago
  Process: 11841 ExecReload=/bin/kill -HUP $MAINPID (code=exited, status=0/SUCCESS)
 Main PID: 1205 (sshd)
    Tasks: 1
   Memory: 1.3M
      CPU: 2.107s
   CGroup: /system.slice/ssh.service
           └─1205 /usr/sbin/sshd -D
```

We can do something like this which is more nicer and readable.

![server-online](https://cloud.githubusercontent.com/assets/622699/26031942/7a1e3324-383a-11e7-8d64-71ae2ab25980.png)
![server-offline](https://cloud.githubusercontent.com/assets/622699/26031943/7ede308a-383a-11e7-9bd6-0e14e9bf5e5a.png)

In-case of distributed it has to be a little different in the error scenario but you get the idea.